docs2/howto.rst

```diff
@@ -99,6 +99,37 @@ This requires some updated pip packages for testing, see https://docs.ros.org/en
 
 Then execute:
 
-```
-colcon test --event-handlers=console_cohesion+ --return-code-on-test-failure --packages-select crazyflie_py
-```
+.. code-block:: bash
+
+    colcon test --event-handlers=console_cohesion+ --return-code-on-test-failure --packages-select crazyflie_py
+
+Collision Avoidance
+-------------------
+
+The official firmware has support for collision avoidance using the Buffered Voronoi Cell algorithm.
+It requires the use of a motion capture system (so that the positions of other drones are known) and can be enabled
+in the `crazyflies.yaml`:
+
+.. code-block:: yaml
+
+    all:
+        firmware_params:
+            colAv:
+                enable: 1
+
+or inside a Python script via:
+
+.. code-block:: python
+
+    swarm = Crazyswarm()
+    allcfs = swarm.allcfs
+    allcfs.setParam("colAv.enable", 1)
+
+Note that the algorithm might require tuning of its hyperparameters. Documention can be found at https://github.com/bitcraze/crazyflie-firmware/blob/dbb9df1137f11d4e7e3771c56d25a7137b5b69cc/src/modules/src/collision_avoidance.c#L348-L428.
```

Comment:
Technically we should have r[epository documentation](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/) on this but I see that it doesn't compile on the web: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/. There is no mention of the ColAv anywhere.... so perhaps for now indeed refer to the raw doc. I've extended the issue related to this: https://github.com/bitcraze/crazyflie-firmware/issues/1323

Reply:
Yes, I looked at the API doc and thought it might be related to colAv not being "core" or something like that. Happy to change it to a more reasonable link in the future.

Reply:
Ah yeah true it is not core (maybe it should be), but it should still render all the parameters. In the doc itself it indicates if they are core or not... so this is clearly a bug on the repository documentation handling. Hope we can fix it soon but your solution now is all fine.